### DISCUSSION TITLE
Server Down.

### DISCUSSION THREAD

**Discussion by Sparks33** [nz-m/SocialEcho#43] [Q&A]
Whenever I run the code, it keeps on saying this:

```Server is down. Please try again later.```

This is what logs in the console:

Proxy error: Could not proxy request /server-status from localhost:3000 to http://127.0.0.1:4000.

**Comment by Phurba-Sherpa** [comment]
You getting errors because your server is down, I followed the below steps to get server up and running
1) npm install, could not install all the packages so deleted the package-lock.json and nodule modules
2) installed packages again
3) set the secret key for access and refresh the token
You are good to go! Enjoy✌
